components/third-party/external-secrets/operator/helmRelease.yaml

```diff
@@ -18,6 +18,8 @@ spec:
     remediation:
       retries: 3
   values:
+    podLabels:
```

Comment:
After deployment, check that these labels are also set on webhook and cert-controller pods, and not only onthe operator pod